pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: rudder_sdk_flutter
-version: 1.0.7
```

Comment:
I would advise against using 1.0.7 because this PR contains breaking changes in the interface such as setAnonymousId -> putAnonymousId

Either the old interface is market as deprecated and it calls the new functions, or it should be a major version, otherwise all the builds of people using this package will break.